app/services/description_generator.rb

```diff
@@ -20,11 +20,12 @@ def initialize(work:)
   def generate
     Cocina::Models::Description.new({
                                       title: title,
-                                      contributor: contributors,
+                                      contributor: ContributorsGenerator.generate(work: work),
                                       subject: keywords,
                                       note: [abstract, citation, contact],
                                       event: [created_date, published_date].compact,
-                                      relatedResource: related_links + related_works
+                                      relatedResource: related_links + related_works,
+                                      form: ContributorsGenerator.form_from_contributors(work: work)
```

Comment:
yes, the contributor information from h2 can trigger `form` elements in the cocina model.  yay!

Reply:
:1st_place_medal: